docs/references/protocol/transactions/types/credentialcreate.md

```diff
@@ -0,0 +1,53 @@
+---
+seo:
+    description: Provisionally issue a credential to a subject account.
+status: not_enabled
+---
+# CredentialCreate
+
+A CredentialCreate transaction creates a credential in the ledger. The issuer of the credential uses this transaction to provisionally issue a credential. The credential is not valid until the subject of the credential accepts it with a [CredentialAccept transaction][].
+
+_(Requires the [Credentials amendment][] {% not-enabled /%})_
+
+## Example CredentialCreate JSON
+
+```json
+{
+    "TransactionType" : "CredentialCreate",
+    "Account": "ra5nK24KXen9AHvsdFTKHSANinZseWnPcX",
+    "Subject": "rsUiUMpnrgxQp24dJYZDhmV4bE3aBtQyt8",
+    "CredentialType": "6D795F63726564656E7469616C",
+    "Fee": "10",
+    "Flags": 0,
+    "Sequence": 234200
+}
+```
+
+
+## CredentialCreate Fields
+
+In addition to the [common fields][], CredentialCreate transactions use the following fields:
+
+| Field            | JSON Type        | [Internal Type][] | Required? | Description |
+|:-----------------|:-----------------|:------------------|:----------|:------------|
+| `Subject` | String - [Address][] | AccountID | Yes | The subject of the credential. |
+| `CredentialType` | String - Hexadecimal | Blob | Yes | Arbitrary data defining the type of credential this entry represents. The minimum length is 1 byte and the maximum length is 64 bytes. |
+| `Expiration` | Number | UInt32 | No | Time after which this credential should be considered expired, in [seconds since the Ripple Epoch][]. |
```

Comment:
```suggestion
| `Expiration` | Number | UInt32 | No | Time after which this credential expires, in [seconds since the Ripple Epoch][]. |
```
Remove the ambiguity, unless there are cases where the expiration time doesn't kick in?